launch.go

```diff
@@ -192,6 +192,12 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 		return fmt.Errorf("Fetching Build ID %d: %v", buildID, err)
 	}
 
+	log.Printf("Fetching Event %d", b.EventID)
+	e, err := api.EventFromID(b.EventID)
```

Comment:
Please use a more meaningful variable name than `e`

Reply:
I'm just staying consistent with other variables in the file/repo.
Having short 1-2 letter vars also seems to be a golang convention:
- http://wordaligned.org/articles/go-for-short-variable-names
- https://github.com/golang/go/wiki/CodeReviewComments#variable-names

If we change this one, we should change them all. Do you want me to change all of them?

Reply:
> If we change this one, we should change them all. Do you want me to change all of them?

Honestly, this code base started off with very limited Golang knowledge. We can either perpetuate to maintain the same low proficiency, or actually improve on the code bases we're working in.

Reply:
Updated the variable names